File: 프로그래머스/0/181896. 첫 번째로 나오는 음수/첫 번째로 나오는 음수.py
def solution(num_list):
    # method1
    r=-1
    for i,n in enumerate(num_list):
        if n<0:
            r=i
            break
    # method2
    # r=([i for i,n in enumerate(num_list) if n<0]or[-1])[0]
    return r

File: 프로그래머스/0/181896. 첫 번째로 나오는 음수/첫 번째로 나오는 음수.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> num_list) {
    int r=-1;
    for(int i=0;i<num_list.size();i++) {
        if(num_list[i]<0) {
            r=i;
            break;
        }
    }
    return r;
}